packages/api/src/schema/api-schema.yaml

```
@@ -2102,7 +2078,18 @@ components:
               type: object
               additionalProperties: false
               properties:
-                $ref: "#/components/schemas/ipfs-file-info/properties"
+                cid:
+                  type: string
+                  description: CID of the file on IPFS
+                url:
+                  readOnly: true
+                  type: string
+                  description: URL with IPFS scheme for the file
+                gatewayUrl:
+                  readOnly: true
+                  type: string
+                  description:
+                    URL to access file via HTTP through an IPFS gateway
```

Actually! I remembered that better than repeating each and every field, we can actually $ref the individual fields and only repeat the list here :)

Can you change the places where you fixed the "inheritance $ref" to be like this instead? suggestion+example:
```suggestion
                cid:
                  $ref: "#/components/schemas/ipfs-file-info/properties/cid"
                url:
                  $ref: "#/components/schemas/ipfs-file-info/properties/url"
                gatewayUrl:
                  $ref: "#/components/schemas/ipfs-file-info/properties/gatewayUrl"
```

---

Yes, it is way better! updated it!